core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -324,7 +326,12 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       }
 
       //yield the things we're going to submit to Cromwell
-      (wdl, wfRecs, wfInputsBatch, wfOpts, wfLabels, wfCollection, collectDosUris(workflowBatch), petUserInfo)
+      val dosUris = collectDosUris(workflowBatch)
+      logger.debug(s"collectDosUris found ${dosUris.size} DOS URIs in batch of size ${workflowBatch.size} for submission ${submissionRec.id}")
+      if(dosUris.nonEmpty && dosUris.size <= 20) { //don't log too many in case it upsets the logger
+        logger.debug(dosUris.toString)
```

Comment:
this log statement should have context... probably should include the workflow id and/or submission id for all log statements

Reply:
also consider user `dosUrs.take(20)` instead of not logging anything at all when there are too many